pkg/rpc/convert.go

```diff
@@ -118,7 +118,7 @@ func ConvertToRpcVulns(vulns []types.DetectedVulnerability) []*common.Vulnerabil
 	return rpcVulns
 }
 
-func ConvertFromRpcResults(rpcResults []*scanner.Result) []report.Result {
+func ConvertFromRpcResults(rpcResults []*scanner.Result, resultType string) []report.Result {
```

Comment:
`rpcResults` has `Type` in the struct, right? Why does it need `resultType` as an argument?

Reply:
You are right. I don't recall why I added this, maybe it was prior to introducing the `Type` type. I have removed it now. Thanks!